tests/test_pytest.py

```diff
@@ -46,7 +48,7 @@ def test_class_level_test_for_pytest(self):
             flexmock_teardown()
 
 
-class TestUnittestClass(test_flexmock.TestFlexmockUnittest):
+class TestUnittestClass(unittest.TestCase):
```

Comment:
Should this one also inherit from `FlexmockTestCase` maybe ?

Reply:
This test just tests that pytest works with `unittest.TestCase` so using `FlexmockTestCase` would be wrong.